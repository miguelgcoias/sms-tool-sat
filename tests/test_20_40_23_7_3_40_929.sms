20
0 5 10 5 1 1 1 1 1
9 22 53 7 4 5 4 3 3 2 1
13 8 37 1 8
28 17 79 1 17
1 10 11 4 3 3 1 3
31 20 71 2 8 12
17 18 53 3 4 7 7
12 22 34 5 2 5 3 5 7
27 8 35 6 2 1 2 1 1 1
23 14 65 5 2 4 3 4 1
37 13 50 4 2 3 5 3
30 21 93 5 6 2 4 5 4
31 22 97 1 22
30 2 32 2 1 1
5 11 16 7 2 1 2 1 1 3 1
20 17 54 2 9 8
24 23 47 5 3 2 7 6 5
15 7 22 7 1 1 1 1 1 1 1
7 23 76 5 7 1 4 4 7
13 15 43 6 3 2 2 2 3 3
0
3 1 18 3
3 18 15 20
7 2 11 8 7 19 14 15
0
5 18 19 14 10 11
2 2 14
1 5
0
8 17 15 11 8 3 16 1 7
2 7 5
8 5 17 10 18 13 6 20 1
7 14 10 11 5 4 20 9
4 3 18 20 5
0
6 19 7 5 8 20 3
1 15
0
3 17 4 9
0
