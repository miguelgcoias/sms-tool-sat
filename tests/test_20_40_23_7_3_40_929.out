4
1 0 1 2 3 4
9 27 29 30 32 33 34
15 5 7 8 10 11 12 15
20 16 21 23 35 37 40
